status/boost_check_library.py

```diff
@@ -83,6 +107,33 @@ def check_organization_include(self):
                 ''',
                 'org-inc-one')
     
+    def check_organization_macros(self):
+        libdir = os.path.join(self.library_dir,'include','boost',self.library_name)
+        if os.path.isdir(libdir):
+            badmacros = {}
+            for root, dirs, files in os.walk(libdir):
+                if os.path.sep + 'detail' + os.path.sep in root or root.endswith(os.path.sep + 'detail'):
+                    continue
+                for item in files:
+                    ext = os.path.splitext(item)
```

Comment:
As you just just `ext[1]` use  `os.path.splitext(item)[1]`. Then "ext" really is the extension.

Reply:
Will fix